### PR TITLE
[Bug 14418] Player does not accept filenames starting with file://

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: engine/src/player-platform.cpp [hunk 1]
```diff
@@ -1384,7 +1384,9 @@ static bool MCPathIsRemoteURL(const char *p_path)
 {
```
Comment (runrevmark) [inline engine/src/player-platform.cpp:1384]: A really minor thing - but MCPathIsRemoteURL is no longer returning true for a 'remote url' - it is just returning true for a URL. If the function isn't used elsewhere (I don't believe it is) then it should probably be renamed 'MCPathIsURL'.

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 09fabd7

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 09fabd7928681c945214c34bdf8e4cbb9e5625ee

**Comment by livecode-vulcan** [comment]
⁉ merge unchanged 09fabd7928681c945214c34bdf8e4cbb9e5625ee → 6401430253ba6c0ae8dceeb5dd4649248a4655c8

Pull request is already fully merged!

**Pull request merged**